recipebox/templates/authoradd.html

```diff
@@ -0,0 +1,5 @@
+<form action="/authoradd/" method="post">
+  {% csrf_token %}
+  {{form.as_p}}
+  <input type="submit" value="Submit">
+</form>
```

Comment:
As a general rule, wanna make sure that you have an empty line at the end of your files. Usually this can be done automatically -- check your editor for a setting about this.